Sources/AIProxy/Fal/FalService.swift

```diff
@@ -46,6 +46,26 @@ public final class FalService {
         return try await self.getResponse(url: responseURL)
     }
 
+    /// Convenience method for creating a `fal-ai/runway-gen3/turbo/image-to-video` video.
+    ///
+    public func createRunwayGen3AlphaVideo(
```

Comment:
Xcode tip: put your cursor in the function signature and tap `cmd-opt-/`. That will give you a docstring template to follow. Take a look at the docstring on `createFastSDXLImage` for what we expect